torch/backends/cuda/__init__.py

```diff
@@ -160,5 +161,21 @@ def preferred_linalg_library(backend: Union[None, str, torch._C._LinalgBackend]
 
     return torch._C._get_linalg_preferred_backend()
 
+def fused_sdp_enabled():
+    r""" Returns whether fused sdp is enabled or not. """
+    return torch._C._get_fused_sdp_enabled()
+
+def set_fused_sdp_enabled(enabled: bool):
+    r""" Enables or disables fused sdp. """
+    torch._C._set_enabled_fused_sdp(enabled)
```

Comment:
nit: there's two verbs and each in a different tense. Why not `set_sdp_use_fused` or `enable_fused_sdp`?